helm/hpcc/templates/roxie.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       - name: {{ $toponame | quote }}
 {{ include "hpcc.addSentinelProbes" $roxie | indent 8 }}
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addResources" (dict "me" $roxie.resources) | indent 8 }}
```

Comment:
This is resources for the toposerver, I think this should pickup a different resource section.
@richardkchapman should there be resource sections like Thor, for topo, server replicas, and channel replicas?

Reply:
toposerver needs minimal resources, like an agentexec stub, and does not need to be configurable

Reply:
ok, so need to generate the stub resources here for topo.
Should there be separate resource sections for server and channel replicas?